docs/samples/specification/management/generated/setup.py

```diff
@@ -6,31 +6,31 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 # coding: utf-8
-
 from setuptools import setup, find_packages
 
-NAME = "azure-mgmt-sample"
-VERSION = "0.1.0"
 
+PACKAGE_NAME = "azure-mgmt-sample"
+version = "0.1.0"
 # To install the library, run the following
 #
 # python setup.py install
 #
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
-
-REQUIRES = ["msrest>=0.6.21", "azure-core<2.0.0,>=1.20.1", "azure-mgmt-core<2.0.0,>=1.2.1"]
-
 setup(
-    name=NAME,
-    version=VERSION,
+    name=PACKAGE_NAME,
+    version=version,
     description="azure-mgmt-sample",
     author_email="",
     url="",
     keywords=["Swagger", "AutoRestHeadTestService"],
-    install_requires=REQUIRES,
     packages=find_packages(),
     include_package_data=True,
+    install_requires=[
+        "msrest>=0.6.21",
+        "azure-core<2.0.0,>=1.20.1",
+        "azure-mgmt-core>=1.3.0,<2.0.0",
```

Comment:
Why do we have azure-core as a dependency at all in mgmt? azure-core should be installed following azure-mgmt-core need

Reply:
Ok that was my bad, I thought it would be better to explicitly state the `azure-core` dependency, but you are right I forgot about azure-mgmt-core specifying it's own min version of azure-core, which could conflict with what we have here

Reply:
Yeah, I think we only need to claim `azure-mgmt-core` in mgmt without `azure-core`.